### ISSUE TITLE
re4_tweaks vs vanilla PRO difficulty (sanity check)

### ISSUE THREAD

**Issue by zoluxsys** [nipkownix/re4_tweaks#319]
### Validation

- [X] Game has been updated to the latest RE4HD release (v1.1: https://www.re4hd.com/?p=9552)
- [X] Data has been [validated with SFV](https://www.re4hd.com/?p=9454) (a guide to using QuickSFV is available at that link, use that with this 1.1 SFV: [BIO4-HDProject1.1-SFV.zip](https://github.com/emoose/re4-research/files/9207370/BIO4-HDProject1.1-SFV.zip))

### Describe your issue here (drag+drop ZIP to attach it)

Hi There,

(Ver 1.7.7.6, 60 FPS, 1.1 HD MOD, XBOX ONE buttons, smaller HUD)
I recently decided to replay RE4 on PRO difficulty (as usual), but I noticed that the game became noticeable harder with re4_tweaks. I can't explain it, feels enemies move and act faster (faster AI logic, less delay between attacks), have a bit more health (maybe different ctrit chance).
I only played TWO versions, original PC release and this HD, and cleared PRO many times before.... Is this supposed to be the real GC PRO difficulty or placebo?

**Comment by LethalPlacebo** [comment]
I think it's probably a placebo...and I don't mean me.

I've finished this game on Pro difficulty on GC, Wii, and PC, but played the most on PC. I don't feel any difference in difficulty with RE4 Tweaks installed.

**Comment by linkthehylian** [comment]
A friendly reminder that Resident Evil 4 is game that was made to be played at **30 fps**. So playing the hardest possible difficulty at double the original frame-rate will definitely make the game feel a bit harder.

It was already confirmed as well that enemies hit much harder at 60 fps than they do at 30 on Professional, dealing twice the normal amount of intended damage.

**Comment by Herlockes** [comment]
I thought re4tweaks fixed those things?

**Comment by LethalPlacebo** [comment]
> I thought re4tweaks fixed those things?

It fixes a lot, but there are certain things still not quite fixed. Leon's hitbox is slightly bigger at 60 FPS and I believe certain enemy attacks do more damage. That being said, these things have never been super big issues for me and the average player would be hard-pressed to notice them.

**Comment by linkthehylian** [comment]
> I thought re4tweaks fixed those things?

Only 60 fps issues related to the physics and QTEs were. (listed in the README)

I'm sure the issues related to Professional will be looked at in the future.

**Comment by zoluxsys** [comment]
Yeah, this is why when you fight U-3 phase 1, if he grabs you it's a almost instantly drains all of your health, even if fully upgraded.

**Comment by nr1995** [comment]
Is it possible that increasing the FOV too much
makes Leon's default aim position slightly off,
or at least not what you're used to,
making headshots feel harder to pull off?
